compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -12619,6 +12623,103 @@ private STNode parseListMatchPatternMemberRhs(SyntaxKind nextTokenKind) {
         }
     }
 
+    /**
+     * Parse mapping match pattern.
+     * <p>
+     *     mapping-match-pattern := { field-match-patterns }
+     *     field-match-patterns :=
```

Comment:
add `<br/>` for line breaks.. otherwise the IDE preview will look cluttered.

Reply:
changed